kfdef/kfctl_ibm_custom_openshift.v1.2.0.yaml

```diff
@@ -0,0 +1,186 @@
+  #       path: stacks/ibm/application/admission-webhook
+  #   name: admission-webhook
+  
+  # Profile controller 
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/profile-control-plane
+  #   name: kubeflow-apps
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift
+    name: kubeflow-apps
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/metadata
+    name: metadata
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/katib
+    name: katib
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/ibm/application/kfp-tekton
+    name: kfp-tekton
+  # Switch the above kfp-tekton to
+  # the below applications if you want to
+  # run KFP with Argo
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/argo
+  #   name: argo
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/kfp-argo
+  #   name: kfp-argo
+  # notebook
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/jupyter-web-app
+    name: jupyter-web-app
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/notebook-controller
+    name: notebook-controller
+  # if the above does not work
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/notebooks
+  #   name: notebooks
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/pytorch-job
+    name: pytorch-job
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: stacks/openshift/application/tf-job
+    name: tf-job
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: metacontroller/base
+    name: metacontroller
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: knative/installs/generic
+    name: knative
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: kfserving/installs/generic
+    name: kfserving
+  # Spartakus is a separate applications so that kfctl can remove it
+  # to disable usage reporting
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/spartakus
+  #   name: spartakus
+  # Optional applications
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/seldon-core-operator
+  #   name: seldon-core-operator
+  # - kustomizeConfig:
+  #     repoRef:
+  #       name: manifests
+  #       path: stacks/ibm/application/spark-operator
+  #   name: spark-operator
+  repos:
+  - name: manifests
+    uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
```

Comment:
For v1.2.0 yaml, usually is recommended to point to the v1.2.0 branch to avoid any upstream changes. If we are using master, then we shouldn't call it v1.2.0 since the master branch is not a stable version.

Reply:
i am making some manifest changes. should i make those pr to 1.2 as well then?

Reply:
i will just call it `kfdef/kfctl_ibm_custom_openshift.yaml` for now. we can create a 1.2.0 yaml if we all agree with the new manifest changes